src/main/antlr4/gov/nasa/pds/api/engineering/lexer/Search.g4

```diff
@@ -6,7 +6,7 @@ group : NOT? LPAREN expression RPAREN ;
 expression : andStatement | orStatement | queryTerm ;
 andStatement : queryTerm (AND queryTerm)+ ;
 orStatement : queryTerm (OR queryTerm)+ ;
-comparison : FIELD operator ( STRINGVAL | NUMBER | VALUE ) ;
+comparison : FIELD operator ( FIELD | NUMBER | STRINGVAL | VALUE ) ;
```

Comment:
I am not sure what is the intention here. I must say I don't understand the VALUE token expression as well. I will approve the merge request so that you can go on.